pulp_rpm/tests/functional/api/test_sync.py

```diff
@@ -128,6 +128,29 @@ def test_sync(self):
         self.assertEqual(latest_version_href, repo.latest_version_href)
         self.assertDictEqual(get_content_summary(repo.to_dict()), RPM_FIXTURE_SUMMARY)
 
+    def test_sync_local(self):
+        """Test syncing from the local filesystem."""
+        import shutil
+
+        if not shutil.which("wget"):
+            unittest.skip("Cannot run file:// sync tests without 'wget' available.")
+
+        cli.Client(self.cfg).run(
+            (
+                "wget",
+                "--recursive",
+                "--no-parent",
+                "--no-host-directories",
+                "--directory-prefix",
+                "/tmp",
+                RPM_UNSIGNED_FIXTURE_URL,
+            )
+        )
+
+        remote = self.remote_api.create(gen_rpm_remote(url="file:///tmp/rpm-unsigned/"))
+
+        self.do_test(remote=remote, mirror=True)
```

Comment:
Missing cleanup - see https://github.com/pulp/pulp_rpm/blob/d87b37747e5c82e789657389657430f7c44524df/pulp_rpm/tests/functional/api/test_sync.py#L116 for example